hotdoc/core/symbols.py

```diff
@@ -62,7 +62,9 @@ def __repr__(self):
     def get_plural_name(cls):
         """Default implementation of the vmethod
         to retrieve Plurial form of the symbol name."""
-        return cls.__tablename__.replace("_", " ").title()
+        if cls.__tablename__:
+            return cls.__tablename__.replace("_", " ").title()
+        return None
```

Comment:
Why? Have you checked the callers?

Reply:
It is legal to return None here, there are precedents. It means "do not create a section for that symbol".